cpp/crypto/Cryptor.cpp

```diff
@@ -124,18 +124,6 @@ static void getRandomBytes(char *const out,
   (void) failure;
 }*/
 
-/**
- * Callback for getting the calling thread's id.
- */
-static void getThreadId(CRYPTO_THREADID *const threadId) noexcept
```

Comment:
Do we have  any other API do to the same stuff?

Reply:
This function was used to set threading callback `CRYPTO_THREADID_set_callback(getThreadId);` which was deprecated and commented out already before this change.